examples/cloudml-churn-prediction/trainer/trainer/model.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorFlow model definition."""
+
+import tensorflow as tf
+
+from trainer import metrics
+
+
+def get_feature_columns(tf_transform_output, exclude_columns=[]):
```

Comment:
per [python style guide 2.12.4](http://go/pystyle#2124-decision)
default params are initialized once on module load so certain defaults are dangerous if they are mutated later on. Even though `exclude_columns` will never be mutated, this is still a best practice.

replace with 
```
def get_feature_columns(tf_transform_output, exclude_columns=None):
  if not exclude_columns:
    exclude_columns = []
```